USER_GUIDE.md

```diff
@@ -211,6 +212,18 @@ To switch back to the Dev App Server v2-alpha (that was default in version < 1.3
 </configuration>
 ```
 
+### How can I pass environment variables to the Dev Appserver (both v1 and v2-alpha)?
+
+You can debug the Dev App Server v1 using the jvmFlags:
```

Comment:
Update this comment?

You can now pass environment variables directly to the Dev App Server.